x-pack/plugins/lens/public/types.ts

```diff
@@ -763,6 +766,10 @@ export type VisualizationDimensionGroupConfig = SharedDimensionProps & {
   /** ID is passed back to visualization. For example, `x` */
   groupId: string;
   accessors: AccessorConfig[];
+  // this is just to show a UI signifier
```

Comment:
// Can we make this comment a bit more specific? Like for example 
```suggestion
// currently used to display non-editable UI dimension trigger in the buckets group when multiple metrics exist for partition chart
```
 - I know it's weirdly specific but I feel like we'll forget what is this only with the comment. I am ok with modifying my proposal if you have better idea :)